__tests__/app.articles.test.js

```diff
@@ -69,4 +69,60 @@ describe("PATCH", () => {
         expect(body.article).toHaveProperty("article_id", 2);
       });
   });
+  test("200: /api/articles/:article_id - returns updated article with decremented vote", () => {
+    const decrementVote = {inc_votes: -100};
+    const article_id = 1;
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .send(decrementVote)
+      .expect(201)
```

Comment:
Should be 200, as you're not creating a new resource, just updating an existing one